firefox-ios/Client/Extensions/URLSessionConfiguration+defaultMPTCP.swift

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension URLSessionConfiguration {
+    static var defaultMPTCP: URLSessionConfiguration {
+        let conf = self.default
+        #if os(iOS)
```

Comment:
This version of Firefox is only built for iOS and there are no plans to built for any other OS in the ecosystem so you can remove this macro.